### DISCUSSION TITLE
Ajout raid de boss

### DISCUSSION THREAD

**Discussion by OrionYp** [Crownicles/Crownicles#2546] [Nouvelles idées]
Explication rapide :
Un joueur invoque le boss, 5 autres (pas forcément de la même guilde) le rejoignent.
Objectif → lui faire le plus de dégâts possibles 
Le boss a vie infinie et tuera les participants les uns après les autres, le combat se termine quand les 6 membres sont morts.

Détails :
Lvl 48 requis pour participer.
Le boss a comme stats : 500 d’attaque, 500 de défense, 500 de vitesse, 10/30 de souffle de base, 5 de regen de souffle. Il n’a pas de ⚡️mais des 💥 (dégâts infligés), qui augmentent au fil du combat.
Il a comme attaques : 
- Attaque simple, - attaque rapide, - attaque vol de souffle, - attaque perçante, - attaque énergique, - attaque sombre.

On peut lancer le combat contre le boss quand on veut et autant de fois qu’on veut à condition de ne pas être sur l’île et de ne pas avoir d’altération.

Récompenses :
Pourquoi pas ajouter des missions secondaires/de campagne « faire un raid » et « faire x dégâts dans un raid »
Un nouveau top selon les dégâts en raid avec les pseudos des 6 participants (et peut-être leurs classes)
Un badge pour avoir participé à un raid ayant fait plus de x (50 000?) dégâts.

Comment se passe un raid :
(J1 = Premier joueur à attaquer)
J1 fait la commande /invocation,
Sous ce message 5 réactions apparaissent 2️⃣3️⃣4️⃣5️⃣6️⃣
Les 5 autres participants cliquent sur la réaction selon l’ordre dans lequel ils souhaitent attaquer.
Lorsque que c’est fait un message apparaît pour le J1 qui dit « voulez-vous lancer le raid avec J2 → pseudo J2, J3 → pseudo J3, etc. » ✅❌

Lorsque le combat se lance :
Tour 1 : le boss attaque J1
Tour 2 : J1 attaque 
Tour 3 : le boss attaque J2
Tour 4 : J2 attaque 
…
Tour 13 : le boss attaque J1
Tour 14 : J1 attaque 
Tour 15 : le boss attaque J2
Tour 16 : J2 attaque 

Quand un joueur meure par le boss, celui-ci enchaîne avec une autre attaque pour le joueur suivant 

Si un joueur mets une altération au boss, cette altération l’affectera à chacun de ses tours.
Si un joueur se mets une propre altération elle affectera chacun de ses tours mais pas les autres joueurs.

Si un joueur fait une attaque en 2 tours le deuxième tour de l’attaque sera fait quand son prochain tour arrivera 

Pour que le combat ne dure pas éternellement, les dégâts du boss augmenteront au fil du combat, pendant la manche 1 (tours de dégâts du boss une fous sur chaque joueur + 1 attaque par joueur) ses dégâts sont multipliés par 0,5 ; manche 2 → 0,75 ; manche 3 → 1 ; manche 4 → 1,25 ; etc.
Donc une attaque normale à 200 dans la manche 1 fera 100 dégâts, puis manche 2 → 150 dégâts, puis manche 3 → 200 dégâts (en imaginant que les stats du boss et des joueurs ne changent pas)

Le combat durerait entre 10 et 20 minutes.

Pour faire le lien avec l’histoire de draftbot on peut dire que les joueurs ont réussi un rituel de l’Ordre des Trois pour invoquer Y'ahnor, la divinité destructrice, afin de tenter de la tuer et empêcher la secte de semer le chaos (enfin je compte sur les rédacteurs pour faire bien mieux que moi ce qui ne devrait pas être difficile 😅)

(Merci à Bananeplantain, Hitori, Evatco, Kiricare, LaMangousteRose, et d’autres pour l’aide)
Évidemment cette proposition n’est pas parfaite donc si vous avez des questions/demandes de précision n’hésitez pas à les poser ici ou sur le général du discord https://discord.com/channels/429765017332613120/433538042901561365, ou sur le post de la suggestion https://discord.com/channels/429765017332613120/1199415351323144376 (n’hésitez pas à me ping @oriony)
Et avant qu’on me fasse la remarque, évidemment que c’est pas pour la V5 ou avant, c’est pour après la V5.

**Comment by Voisin1234** [comment]
J'ai pas tout lu, mais juste pour le titre je valide plus plus

**Comment by Ntalcme** [comment]
J'aime beaucoup l'idée même si à mon avis y a peut-être quelques points à clarifier mais l'idée en elle-même est géniale.

**Reply by OrionYp** [reply]
Je suis là pour répondre aux questions sur les points à clarifier n'hésite pas ^^.
Enfin c'est pas urgent on a quelques mois avant de savoir si c'est accepté

**Comment by StrangerIM** [comment]
Pas de récompense si pas de mission et badge déjà obtenu ?

**Reply by OrionYp** [reply]
Juste pouvoir monter dans le classement oui,
Mais on peut mettre des missions quotidiennes en rapport avec ce système.

S'il y avait des récompenses on devrait limiter les tentatives par semaine ou devoir payer quelque chose pour faire un combat (sinon on aurait une source infinie de récompenses)

**Reply by OrionYp** [reply]
Et comme la difficulté du boss n'augmente pas avec le niveau des joueurs ça veut dire que tout les mois quand les joueurs montent d'un lvl ils peuvent faire plus de dégâts et monter plus haut dans le classement. Donc le classement ne restera jamais le même

**Comment by OrionYp** [comment]
Réponses aux questions : 

Q0: Je te fais remarquer que le principal intérêt de l'énergique est de régénérer son énergie ça ne fait pas sens de la donner au boss (sauf si on soustrait l'énergie récupérée du total de dégâts reçus) puisqu'il a de l'énergie illimité.C'est un IA donc il ne réfléchit pas forcément en combo etc mais je crains que le combat soit juste du spam de percante ou un boss mage qui alterne vol de souffle/sombre (ou encore plus OP vol/énergique).Ça risque d'être frustrant de donner une attaque qui baisse la déf(perçante),une autre la vitesse(vol de souffle)(rendant la rapide toujours plus puissante),une dernière l'attaque (sombre) du joueur ,résultat à la fin on prend très cher et on fait peu de dégâts (comme quand on affronte le golem t30).Bref un peu d'équilibrage à réfléchir.


R0: le boss baisse les stats des joueurs mais n’oublie pas qu’il y a 6 joueurs au début donc il peut baisser la stat d’un joueur 1/12 tours
Alors que les joueurs peuvent baisser les stats du boss 1/2 tours, donc ça s’équilibre.
Pour l'attaque énergique j'avoue que ça n'a pas de sens tu as raison, on peut l'enlever.



Q1: Est-ce que ça marche comme un vrai combat ou un amical ? ->énergie des joueurs et potions consommés ?
Mon avis balance entre oui c'est du PVE et non  c'est en illimité, quand on veut etc. (plus vers non quand même)


R1: Finalement je vais ajouter la proposition d’elec dans le post sur l’offi
On peut faire des amicaux sans consommation de potion et d’énergie quand on veut 
Et pour faire un combat avec consommation d’énergie et potions pour être dans le classement 
Pour ce « vrai » combat il faut que J1 récupère x parchemins en mini évents
Balance parfaite entre oui et non.

Q2: Le boss commence toujours ? Parce que 500 de vitesse y a des gens (des canos surtout) qui ont plus (dans ce cas voir pour chaque doublette de tours (=attaque boss +attaque joueur) dans quel ordre selon leur vitesse et donc le boss n'a pas forcément les tours impairs ,je m'explique :le J1/J2/J4 sont plus rapides que le boss -> J1-Boss vs J1 puis J2-boss vs J2 puis Boss vs J3 -J3...)


R2: Je pense que c’est bien de faire « le boss attaque → le joueur attaque → changement de joueur » et on recommence 
On va moins se perdre que « joueur attaque → boss attaque → boss attaque un autre joueur → l’autre joueur attaque » 



Q3: Pendant un combat, quand on ne commence pas on a du souffle en plus ,le reçoit-on ici ?


R3: j’aimerais bien si possible que :
Tour 1 → pas de regen pour le boss
Tour 2 → regen normale pour J1
Et après que de la regen normale

**Comment by OrionYp** [comment]
Suggestion d'ajout au système de raid : 
 

Des fragments de parchemins :
On pourrait faire le raid en amical, pas de récompenses et on peut le faire autant de fois qu’on veut, sans consommation de potion ou d’énergie.
Ou alors utiliser des fragments de parchemins, on pourrait les obtenir en mini évent/évent, et lorsque qu’on en a 3 on peut lancer une invocation du dieu (seul le premier joueur paye les 3 parchemins, les autres le rejoigne gratuitement) 
Et ce raid classé permettrait d’avoir une place dans le classement et de gagner quelques récompenses (le badge cité ci-dessus, et de l’or/ de l’exp)
Pour le stockage de fragments je ne sais pas de combien la limite devrait être.

(Merci à Electek et Bananeplantain pour les questions et suggestions)
